osl/preprocessing/batch.py

```diff
@@ -259,6 +260,17 @@ def load_config(config):
             )
 
     return config
+   
+   
+def get_config_from_fif(data):
+
+    config_list = re.findall('%% config start %%(.*?)%% config end %%', data.info['description'], flags=re.DOTALL)
+    config=[]
+    for config_text in config_list:
+    config.append(load_config(config_text))
```

Comment:
So apparently this doesn't work. I added the loop in case there are multiple occurrences of the config in the description. So config_list is a list of strings. Strangely, the loop will run into problems: 
```line 237: KeyError: 'event_codes must be passed in config if we are finding events.' ```
but if you would run the same thing without the loop, e.g. `load_config(config_list[0])` it runs smoothly. `config_list[0]==config_text` is True, so I have no idea what's happening here..

Reply:
I think it's working now!